### DISCUSSION TITLE
Complejidad

### DISCUSSION THREAD

**Discussion by SantiagoOsorio3** [IIC2133-PUC/2024-2#96] [Tarea 1] [answered]
Hola, queria preguntar que pasa si es que uno en una funcion no cumple con la complejidad? esa fucnion entera estará mala o se descuenta un porcentaje...

**Comment by tomastrivino** [comment, marked as answer]
Hola, 
los tests están diseñados para no ser exitosos si es que no se cumple con la complejidad solicitada. Esto implica que se tendría 0 puntos (de 1) en los tests correspondientes en los que no cumplas con la complejidad.
